File: app/src/main/java/com/zafaris/elevenplusvocab/ui/SplashScreenActivity.java
package com.zafaris.elevenplusvocab.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.zafaris.elevenplusvocab.R;
import com.zafaris.elevenplusvocab.ui.main.MainActivity;

import gr.net.maroulis.library.EasySplashScreen;

public class SplashScreenActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EasySplashScreen config = new EasySplashScreen(SplashScreenActivity.this)
                .withFullScreen()
                .withTargetActivity(MainActivity.class)
                .withSplashTimeOut(3000)
                .withBackgroundResource(R.mipmap.ic_launcher_background)
                //.withHeaderText("Header")
                //.withFooterText("Zafaris Developers")
                //.withBeforeLogoText("Before Logo Text")
                //.withAfterLogoText("After Logo Text")
                .withLogo(R.mipmap.ic_launcher_foreground);

        config.getLogo().getLayoutParams().height = 1000;
        config.getLogo().getLayoutParams().width = 1000;
        //config.getHeaderTextView().setTextColor(Color.WHITE);
        //config.getFooterTextView().setTextColor(Color.WHITE);
        //config.getBeforeLogoTextView().setTextColor(Color.WHITE);
        //config.getAfterLogoTextView().setTextColor(Color.WHITE);

        View easySplashScreen = config.create();
        setContentView(easySplashScreen);
    }
}

File: app/src/main/java/com/zafaris/elevenplusvocab/ui/SplashScreenActivity.kt
package com.zafaris.elevenplusvocab.ui

import android.content.Intent
import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.view.View
import gr.net.maroulis.library.EasySplashScreen
import com.zafaris.elevenplusvocab.ui.main.MainActivity
import com.zafaris.elevenplusvocab.R

class SplashScreenActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)

        window.statusBarColor = getColor(R.color.colorBlueStatus)
        supportActionBar?.hide()

        Handler().postDelayed({
            val intent = Intent(this@SplashScreenActivity, MainActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000)

    }
}